vispy/util/fetching.py

```diff
@@ -90,7 +96,10 @@ def get_testing_file(fname, directory=None, force_download=False):
     fname : str
         The path to the file on the local system.
     """
-    _url_root = 'https://github.com/vispy/test-data/raw/master/'
+    global test_data_commit_id
```

Comment:
Actually now that I think about it, this code will only be around for a short while in order to bootstrap the new image tests in place. #870 will remove it and replace with a git-based system.